go-controller/pkg/network-controller-manager/node_network_controller_manager.go

```diff
@@ -175,7 +176,7 @@ func (ncm *nodeNetworkControllerManager) Stop() {
 func (ncm *nodeNetworkControllerManager) checkForStaleOVSRepresentorInterfaces() {
 	// Get all ovn-kuberntes Pod interfaces. these are OVS interfaces that have their external_ids:sandbox set.
 	out, stderr, err := util.RunOVSVsctl("--columns=name,external_ids", "--data=bare", "--no-headings",
-		"--format=csv", "find", "Interface", "external_ids:sandbox!=\"\"", "external_ids:vf-netdev-name!=\"\"")
+		"--format=csv", "find", "Interface", "external_ids:sandbox!=\"\"")
```

Comment:
This `checkForStaleOVSRepresentorInterfaces` also runs in normal clusters w/o hardware offload device, every 1 min. We had an improvement to reduce the memory usage when running ovs-vsctl : https://github.com/ovn-org/ovn-kubernetes/pull/3054, would it make sense to only query data with `external_ids:vf-netdev-name` and `external_ids:netdev-name` in this cmd instead of using regexp match later?

Reply:
I haven't found logical `or` for external_ids. If it exists, that'll be great.

Reply:
I didn't find a logical `or` either. would it make sense to have two separate operations here, one for netdev-name, the other for vf-netdev-name? I noticed @cathy-zhou has a comment to change the netdev-name back to vf-netdev-name, maybe this can be done in that separate PR?